static/dashboard.js

```diff
@@ -15,7 +15,7 @@ xhr.onreadystatechange = function() {
         var apps_html = ""
         for (app of response.apps) {
             console.log(app);
-            apps_html += "<div class=\"col-sm-2\"><div class=\"card fort-tile\"><a href=\""+ app.url +"\" target=\"_blank\"><img src=\""+ app.icon +"\" class=\"fort-tileImg float-left\" alt=\"\"></a><h5 class=\"card-title\"><a class=\"fort-tileName text-break text-center\"  target=\"_blank\" href=\""+ app.url +"\">"+app.name+"</a></h5></div></div>"
+            apps_html += "<div class=\"col-sm-2\"><div class=\"card fort-tile\"><a href=\""+ app.url +"\" target=\"_blank\"><img src=\"/dashboard"+ app.icon +"\" class=\"fort-tileImg float-left\" alt=\"\"></a><h5 class=\"card-title\"><a class=\"fort-tileName text-break text-center\"  target=\"_blank\" href=\""+ app.url +"\">"+app.name+"</a></h5></div></div>"
```

Comment:
This shouldn't be hardcoded, but since it's a simple change I will merge it and fix myself.